diff -uprN eris-0.9.8.old/Eris/BaseConnection.cpp eris-0.9.8/Eris/BaseConnection.cpp
--- eris-0.9.8.old/Eris/BaseConnection.cpp	2003-09-29 16:02:05.000000000 -0700
+++ eris-0.9.8/Eris/BaseConnection.cpp	2003-09-29 16:02:47.000000000 -0700
@@ -145,7 +145,7 @@ void BaseConnection::recv()
 void BaseConnection::nonblockingConnect()
 {
 	assert(_stream);
-    if (!_stream->is_ready())
+    if (!_stream->isReady())
 		return;
 
     if(!_stream->is_open()) {
