diff -uprN eris-0.9.8.old/configure eris-0.9.8/configure
--- eris-0.9.8.old/configure	2003-09-29 15:55:20.000000000 -0700
+++ eris-0.9.8/configure	2003-09-29 15:56:09.000000000 -0700
@@ -9181,23 +9181,23 @@ fi
   else
      PKG_CONFIG_MIN_VERSION=0.9.0
      if $PKG_CONFIG --atleast-pkgconfig-version $PKG_CONFIG_MIN_VERSION; then
-        echo "$as_me:$LINENO: checking for skstream-0.2 atlascpp-0.4 wfmath-0.2" >&5
-echo $ECHO_N "checking for skstream-0.2 atlascpp-0.4 wfmath-0.2... $ECHO_C" >&6
+        echo "$as_me:$LINENO: checking for skstream-0.3 atlascpp-0.4 wfmath-0.2" >&5
+echo $ECHO_N "checking for skstream-0.3 atlascpp-0.4 wfmath-0.2... $ECHO_C" >&6
 
-        if $PKG_CONFIG --exists "skstream-0.2 atlascpp-0.4 wfmath-0.2" ; then
+        if $PKG_CONFIG --exists "skstream-0.3 atlascpp-0.4 wfmath-0.2" ; then
             echo "$as_me:$LINENO: result: yes" >&5
 echo "${ECHO_T}yes" >&6
             succeeded=yes
 
             echo "$as_me:$LINENO: checking WF_CFLAGS" >&5
 echo $ECHO_N "checking WF_CFLAGS... $ECHO_C" >&6
-            WF_CFLAGS=`$PKG_CONFIG --cflags "skstream-0.2 atlascpp-0.4 wfmath-0.2"`
+            WF_CFLAGS=`$PKG_CONFIG --cflags "skstream-0.3 atlascpp-0.4 wfmath-0.2"`
             echo "$as_me:$LINENO: result: $WF_CFLAGS" >&5
 echo "${ECHO_T}$WF_CFLAGS" >&6
 
             echo "$as_me:$LINENO: checking WF_LIBS" >&5
 echo $ECHO_N "checking WF_LIBS... $ECHO_C" >&6
-            WF_LIBS=`$PKG_CONFIG --libs "skstream-0.2 atlascpp-0.4 wfmath-0.2"`
+            WF_LIBS=`$PKG_CONFIG --libs "skstream-0.3 atlascpp-0.4 wfmath-0.2"`
             echo "$as_me:$LINENO: result: $WF_LIBS" >&5
 echo "${ECHO_T}$WF_LIBS" >&6
         else
@@ -9205,7 +9205,7 @@ echo "${ECHO_T}$WF_LIBS" >&6
             WF_LIBS=""
             ## If we have a custom action on failure, don't print errors, but
             ## do set a variable so people can do so.
-            WF_PKG_ERRORS=`$PKG_CONFIG --errors-to-stdout --print-errors "skstream-0.2 atlascpp-0.4 wfmath-0.2"`
+            WF_PKG_ERRORS=`$PKG_CONFIG --errors-to-stdout --print-errors "skstream-0.3 atlascpp-0.4 wfmath-0.2"`
 
         fi
 
@@ -9221,7 +9221,7 @@ echo "${ECHO_T}$WF_LIBS" >&6
 
         CPPFLAGS="$CPPFLAGS $WF_CFLAGS"
         LDFLAGS="$LDFLAGS $WF_LIBS"
-        ERIS_REQUIRES="$ERIS_REQUIRES skstream-0.2 atlascpp-0.4 wfmath-0.2"
+        ERIS_REQUIRES="$ERIS_REQUIRES skstream-0.3 atlascpp-0.4 wfmath-0.2"
 
   else
      { { echo "$as_me:$LINENO: error: Couldn't find the required WorldForge libraries" >&5
diff -uprN eris-0.9.8.old/configure.in eris-0.9.8/configure.in
--- eris-0.9.8.old/configure.in	2003-09-29 15:55:20.000000000 -0700
+++ eris-0.9.8/configure.in	2003-09-29 15:55:44.000000000 -0700
@@ -50,11 +50,11 @@ PKG_CHECK_MODULES(SIGC, sigc++-1.2 >= 1.
 AC_MSG_ERROR(Couldn't find libsigc++ config [or config failed]))
 
 # check for all the WorldForge libs we use
-PKG_CHECK_MODULES(WF, skstream-0.2 atlascpp-0.4 wfmath-0.2,
+PKG_CHECK_MODULES(WF, skstream-0.3 atlascpp-0.4 wfmath-0.2,
     [
         CPPFLAGS="$CPPFLAGS $WF_CFLAGS"
         LDFLAGS="$LDFLAGS $WF_LIBS"
-        ERIS_REQUIRES="$ERIS_REQUIRES skstream-0.2 atlascpp-0.4 wfmath-0.2"
+        ERIS_REQUIRES="$ERIS_REQUIRES skstream-0.3 atlascpp-0.4 wfmath-0.2"
     ],
     AC_MSG_ERROR(Couldn't find the required WorldForge libraries))
 
