diff -urpN fortune-mod-9708.old/datfiles/zippy fortune-mod-9708/datfiles/zippy
--- fortune-mod-9708.old/datfiles/zippy	2004-06-17 09:02:13.145096728 -0700
+++ fortune-mod-9708/datfiles/zippy	2004-06-17 09:04:50.558166304 -0700
@@ -949,7 +949,7 @@ empty OIL DRUMS ...
 %
 Should I do my BOBBIE VINTON medley?
 %
-Should I get locked in the PRINCICAL'S OFFICE today -- or have a VASECTOMY??
+Should I get locked in the PRINCIPAL'S OFFICE today -- or have a VASECTOMY??
 %
 Should I start with the time I SWITCHED personalities with a BEATNIK
 hair stylist or my failure to refer five TEENAGERS to a good OCULIST?
