diff -Naur tong-1.0.orig/Makefile tong-1.0/Makefile
--- tong-1.0.orig/Makefile	2005-01-06 19:34:56.530029016 -0800
+++ tong-1.0/Makefile	2005-01-06 19:35:33.889349536 -0800
@@ -16,13 +16,13 @@
 #builds all, builds the target
 
 all:	$(OBJS)
-	$(CC) $(SDL_LDFLAGS) $(MIXER_FLAGS) $(IMG_FLAGS) $(OBJS) -o $(TARGET)
+	$(CC) $(CXXFLAGS) $(SDL_LDFLAGS) $(MIXER_FLAGS) $(IMG_FLAGS) $(OBJS) -o $(TARGET)
 
 %.o: %.cpp
-	$(CC) $(SDL_CFLAGS) -c -o $@ $<
+	$(CC) $(CXXFLAGS) $(SDL_CFLAGS) -c -o $@ $<
 
 tong.o: $(SDL_SOURCES)
-	$(CC) $(SDL_CFLAGS) -c -o $@ $<
+	$(CC) $(CXXFLAGS) $(SDL_CFLAGS) -c -o $@ $<
 
 # clean out the mess
 clean:
