? mercator_gcc43.patch
Index: Mercator/AreaShader.cpp
===================================================================
RCS file: /home/cvspsrv/worldforge/forge/libs/mercator/Mercator/AreaShader.cpp,v
retrieving revision 1.11
diff -u -p -r1.11 AreaShader.cpp
--- Mercator/AreaShader.cpp	20 Sep 2007 18:24:35 -0000	1.11
+++ Mercator/AreaShader.cpp	20 Apr 2008 00:42:52 -0000
@@ -14,6 +14,7 @@
 
 #include <set>
 #include <iostream>
+#include <algorithm>
 
 namespace Mercator
 {
Index: Mercator/AreaShader.h
===================================================================
RCS file: /home/cvspsrv/worldforge/forge/libs/mercator/Mercator/AreaShader.h,v
retrieving revision 1.5
diff -u -p -r1.5 AreaShader.h
--- Mercator/AreaShader.h	20 Sep 2007 18:24:35 -0000	1.5
+++ Mercator/AreaShader.h	20 Apr 2008 00:42:52 -0000
@@ -26,7 +26,7 @@ public:
     virtual bool checkIntersect(const Segment &) const;
 private:
     /// helper to shader a single area into the surface
-    void shadeArea(Surface& s, const Area* const s) const;
+    void shadeArea(Surface& s, const Area* const ar) const;
     
     /// The layer number.
     int m_layer;
Index: Mercator/RandCache.h
===================================================================
RCS file: /home/cvspsrv/worldforge/forge/libs/mercator/Mercator/RandCache.h,v
retrieving revision 1.5
diff -u -p -r1.5 RandCache.h
--- Mercator/RandCache.h	20 Sep 2007 18:24:35 -0000	1.5
+++ Mercator/RandCache.h	20 Apr 2008 00:42:53 -0000
@@ -6,6 +6,7 @@
 #define MERCATOR_RANDCACHE_H
 
 #include <vector>
+#include <cstdlib>
 #include <wfmath/MersenneTwister.h>
 
 // construct with something like:


