diff -uprN janus-0.4.6.old/bindings/gtk/pushbutton.h janus-0.4.6/bindings/gtk/pushbutton.h
--- janus-0.4.6.old/bindings/gtk/pushbutton.h	2003-10-05 13:34:59.000000000 -0700
+++ janus-0.4.6/bindings/gtk/pushbutton.h	2003-10-05 13:41:47.000000000 -0700
@@ -23,6 +23,7 @@
 #ifndef _PUSHBUTTON_H_
 #define _PUSHBUTTON_H_
 
+#include <assert.h>
 #include "widgetbase.h"
 namespace Janus {
 
diff -uprN janus-0.4.6.old/bindings/libuta/checkbox.cpp janus-0.4.6/bindings/libuta/checkbox.cpp
--- janus-0.4.6.old/bindings/libuta/checkbox.cpp	2003-10-05 13:27:39.000000000 -0700
+++ janus-0.4.6/bindings/libuta/checkbox.cpp	2003-10-05 13:41:47.000000000 -0700
@@ -1,3 +1,4 @@
+#include <assert.h>
 #include "checkbox.h"
 
 #include <uta/resources.h>
diff -uprN janus-0.4.6.old/bindings/libuta/dialog.cpp janus-0.4.6/bindings/libuta/dialog.cpp
--- janus-0.4.6.old/bindings/libuta/dialog.cpp	2003-10-05 13:27:39.000000000 -0700
+++ janus-0.4.6/bindings/libuta/dialog.cpp	2003-10-05 13:41:47.000000000 -0700
@@ -1,3 +1,4 @@
+#include <assert.h>
 #include "dialog.h"
 
 
diff -uprN janus-0.4.6.old/bindings/libuta/drawable.cpp janus-0.4.6/bindings/libuta/drawable.cpp
--- janus-0.4.6.old/bindings/libuta/drawable.cpp	2003-10-05 13:27:39.000000000 -0700
+++ janus-0.4.6/bindings/libuta/drawable.cpp	2003-10-05 13:42:03.000000000 -0700
@@ -1,3 +1,4 @@
+#include <assert.h>
 #include "drawable.h"
 
 #include <uta/painter.h>
diff -uprN janus-0.4.6.old/bindings/libuta/label.cpp janus-0.4.6/bindings/libuta/label.cpp
--- janus-0.4.6.old/bindings/libuta/label.cpp	2003-10-05 13:27:39.000000000 -0700
+++ janus-0.4.6/bindings/libuta/label.cpp	2003-10-05 13:41:47.000000000 -0700
@@ -1,3 +1,4 @@
+#include <assert.h>
 #include "label.h"
 #include <uta/multilineedit.h>
 #include <uta/resources.h>
diff -uprN janus-0.4.6.old/bindings/libuta/lineedit.cpp janus-0.4.6/bindings/libuta/lineedit.cpp
--- janus-0.4.6.old/bindings/libuta/lineedit.cpp	2003-10-05 13:27:39.000000000 -0700
+++ janus-0.4.6/bindings/libuta/lineedit.cpp	2003-10-05 13:41:47.000000000 -0700
@@ -1,3 +1,4 @@
+#include <assert.h>
 #include "lineedit.h"
 #include <uta/lineedit.h>
 #include <uta/resources.h>
diff -uprN janus-0.4.6.old/bindings/libuta/listbox.cpp janus-0.4.6/bindings/libuta/listbox.cpp
--- janus-0.4.6.old/bindings/libuta/listbox.cpp	2003-10-05 13:27:39.000000000 -0700
+++ janus-0.4.6/bindings/libuta/listbox.cpp	2003-10-05 13:41:47.000000000 -0700
@@ -1,3 +1,4 @@
+#include <assert.h>
 #include "listbox.h"
 #include <uta/listbox.h>
 
diff -uprN janus-0.4.6.old/bindings/libuta/pushbutton.cpp janus-0.4.6/bindings/libuta/pushbutton.cpp
--- janus-0.4.6.old/bindings/libuta/pushbutton.cpp	2003-10-05 13:27:39.000000000 -0700
+++ janus-0.4.6/bindings/libuta/pushbutton.cpp	2003-10-05 13:41:47.000000000 -0700
@@ -1,3 +1,4 @@
+#include <assert.h>
 #include "pushbutton.h"
 #include <uta/pushbutton.h>
 #include <uta/resources.h>
diff -uprN janus-0.4.6.old/bindings/libuta/slider.cpp janus-0.4.6/bindings/libuta/slider.cpp
--- janus-0.4.6.old/bindings/libuta/slider.cpp	2003-10-05 13:27:39.000000000 -0700
+++ janus-0.4.6/bindings/libuta/slider.cpp	2003-10-05 13:41:47.000000000 -0700
@@ -1,3 +1,4 @@
+#include <assert.h>
 #include "slider.h"
 #include <uta/slider.h>
 #include <uta/resources.h>
diff -uprN janus-0.4.6.old/bindings/libuta/widgetbase.cpp janus-0.4.6/bindings/libuta/widgetbase.cpp
--- janus-0.4.6.old/bindings/libuta/widgetbase.cpp	2003-10-05 13:27:39.000000000 -0700
+++ janus-0.4.6/bindings/libuta/widgetbase.cpp	2003-10-05 13:41:47.000000000 -0700
@@ -1,3 +1,4 @@
+#include <assert.h>
 #include "widgetbase.h"
 
 #include <uta/application.h>
diff -uprN janus-0.4.6.old/bindings/libuta/widget.cpp janus-0.4.6/bindings/libuta/widget.cpp
--- janus-0.4.6.old/bindings/libuta/widget.cpp	2003-10-05 13:27:39.000000000 -0700
+++ janus-0.4.6/bindings/libuta/widget.cpp	2003-10-05 13:41:47.000000000 -0700
@@ -1,3 +1,4 @@
+#include <assert.h>
 #include "widget.h"
 
 using namespace Janus;
