diff -uraN stratagus-2.2.4/Makefile b/Makefile
--- stratagus-2.2.4/Makefile	2007-03-04 14:12:34.000000000 -0800
+++ b/Makefile	2008-02-14 05:41:05.000000000 -0800
@@ -310,11 +310,11 @@
 install:	all install-stratagus
 
 install-stratagus:
-	install -m 755 stratagus $(PREFIX)/bin
+	install -m 755 stratagus $(PREFIX)/games
 	@echo installation of stratagus complete
 
 uninstall:
 	@echo uninstalling stratagus and stratagus tools
-	rm $(PREFIX)/bin/stratagus
+	rm $(PREFIX)/games/stratagus
 	@echo uninstallation of stratagus complete
 
