diff -Nur monkey-bubble-0.1.9/src/ui/eggcellrendererkeys.h monkey-bubble-0.1.9-fixed/src/ui/eggcellrendererkeys.h
--- monkey-bubble-0.1.9/src/ui/eggcellrendererkeys.h	2004-01-30 15:57:55.000000000 +0100
+++ monkey-bubble-0.1.9-fixed/src/ui/eggcellrendererkeys.h	2004-06-23 13:56:17.481182312 +0200
@@ -70,10 +70,10 @@
 
 void             egg_cell_renderer_keys_set_accelerator (EggCellRendererKeys     *keys,
 							 guint                    keyval,
-							 EggVirtualModifierType   mask);
+							 GdkModifierType   mask);
 void             egg_cell_renderer_keys_get_accelerator (EggCellRendererKeys     *keys,
 							 guint                   *keyval,
-							 EggVirtualModifierType  *mask);
+							 GdkModifierType  *mask);
 void             egg_cell_renderer_keys_set_accel_mode  (EggCellRendererKeys     *keys,
 							 EggCellRendererKeysMode  accel_mode);
