diff -Naur cyphesis-0.5.15.orig/common/Database.cpp cyphesis-0.5.15/common/Database.cpp
--- cyphesis-0.5.15.orig/common/Database.cpp	2008-04-26 13:17:57.000000000 -0700
+++ cyphesis-0.5.15/common/Database.cpp	2008-04-26 13:18:09.000000000 -0700
@@ -34,6 +34,8 @@
 #include <iostream>
 #include <sstream>
 
+#include <cstring>
+#include <cstdlib>
 #include <cassert>
 
 using Atlas::Message::Element;
diff -Naur cyphesis-0.5.15.orig/common/globals.cpp cyphesis-0.5.15/common/globals.cpp
--- cyphesis-0.5.15.orig/common/globals.cpp	2008-04-26 13:17:57.000000000 -0700
+++ cyphesis-0.5.15/common/globals.cpp	2008-04-26 13:18:09.000000000 -0700
@@ -36,6 +36,7 @@
 
 #include <sys/stat.h>
 
+#include <cstring>
 #include <cassert>
 
 const char * CYPHESIS = "cyphesis";
diff -Naur cyphesis-0.5.15.orig/common/id.cpp cyphesis-0.5.15/common/id.cpp
--- cyphesis-0.5.15.orig/common/id.cpp	2008-04-26 13:17:57.000000000 -0700
+++ cyphesis-0.5.15/common/id.cpp	2008-04-26 13:18:09.000000000 -0700
@@ -22,6 +22,7 @@
 #include "common/compose.hpp"
 
 #include <cassert>
+#include <cstdlib>
 
 long integerId(const std::string & id)
 {
diff -Naur cyphesis-0.5.15.orig/common/log.cpp cyphesis-0.5.15/common/log.cpp
--- cyphesis-0.5.15.orig/common/log.cpp	2008-04-26 13:17:57.000000000 -0700
+++ cyphesis-0.5.15/common/log.cpp	2008-04-26 13:18:09.000000000 -0700
@@ -27,6 +27,7 @@
 
 #include <iostream>
 #include <fstream>
+#include <cstring>
 
 extern "C" {
 #ifdef HAVE_SYSLOG_H
diff -Naur cyphesis-0.5.15.orig/common/types.h cyphesis-0.5.15/common/types.h
--- cyphesis-0.5.15.orig/common/types.h	2008-04-26 13:17:57.000000000 -0700
+++ cyphesis-0.5.15/common/types.h	2008-04-26 13:19:04.000000000 -0700
@@ -25,6 +25,7 @@
 #include <set>
 #include <map>
 #include <vector>
+#include <string>
 
 class BaseEntity;
 class Entity;
diff -Naur cyphesis-0.5.15.orig/modules/DateTime.cpp cyphesis-0.5.15/modules/DateTime.cpp
--- cyphesis-0.5.15.orig/modules/DateTime.cpp	2008-04-26 13:17:57.000000000 -0700
+++ cyphesis-0.5.15/modules/DateTime.cpp	2008-04-26 13:18:09.000000000 -0700
@@ -20,6 +20,7 @@
 #include "DateTime.h"
 
 #include <stdio.h>
+#include <cstdlib>
 
 // date_pat=re.compile("^|[-:]|\s+");
 
diff -Naur cyphesis-0.5.15.orig/rulesets/Container.h cyphesis-0.5.15/rulesets/Container.h
--- cyphesis-0.5.15.orig/rulesets/Container.h	2008-04-26 13:17:57.000000000 -0700
+++ cyphesis-0.5.15/rulesets/Container.h	2008-04-26 13:18:09.000000000 -0700
@@ -25,6 +25,7 @@
 #include <set>
 #include <map>
 #include <vector>
+#include <string>
 
 class Entity;
 
diff -Naur cyphesis-0.5.15.orig/rulesets/Statistics.h cyphesis-0.5.15/rulesets/Statistics.h
--- cyphesis-0.5.15.orig/rulesets/Statistics.h	2008-04-26 13:17:56.000000000 -0700
+++ cyphesis-0.5.15/rulesets/Statistics.h	2008-04-26 13:18:52.000000000 -0700
@@ -22,6 +22,8 @@
 
 #include "common/types.h"
 
+#include <cstring>
+
 class ArithmeticScript;
 class Character;
 
diff -Naur cyphesis-0.5.15.orig/server/CommListener.cpp cyphesis-0.5.15/server/CommListener.cpp
--- cyphesis-0.5.15.orig/server/CommListener.cpp	2008-04-26 13:17:57.000000000 -0700
+++ cyphesis-0.5.15/server/CommListener.cpp	2008-04-26 13:18:09.000000000 -0700
@@ -35,6 +35,7 @@
 #include <skstream/skstream.h>
 
 #include <iostream>
+#include <cstring>
 
 #ifdef HAVE_ARPA_INET_H
 #include <arpa/inet.h>
diff -Naur cyphesis-0.5.15.orig/server/CommServer.cpp cyphesis-0.5.15/server/CommServer.cpp
--- cyphesis-0.5.15.orig/server/CommServer.cpp	2008-04-26 13:17:57.000000000 -0700
+++ cyphesis-0.5.15/server/CommServer.cpp	2008-04-26 13:18:09.000000000 -0700
@@ -35,6 +35,7 @@
 
 #include <skstream/sksocket.h>
 
+#include <cstring>
 #include <iostream>
 
 extern "C" {
diff -Naur cyphesis-0.5.15.orig/tools/cyconfig.cpp cyphesis-0.5.15/tools/cyconfig.cpp
--- cyphesis-0.5.15.orig/tools/cyconfig.cpp	2008-04-26 13:17:56.000000000 -0700
+++ cyphesis-0.5.15/tools/cyconfig.cpp	2008-04-26 13:18:09.000000000 -0700
@@ -33,6 +33,8 @@
 
 #include <varconf/config.h>
 
+#include <cstring>
+#include <cstdlib>
 #include <cassert>
 
 static int install(int argc, char ** argv)
diff -Naur cyphesis-0.5.15.orig/tools/cyconvertrules.cpp cyphesis-0.5.15/tools/cyconvertrules.cpp
--- cyphesis-0.5.15.orig/tools/cyconvertrules.cpp	2008-04-26 13:17:56.000000000 -0700
+++ cyphesis-0.5.15/tools/cyconvertrules.cpp	2008-04-26 13:18:09.000000000 -0700
@@ -35,6 +35,7 @@
 #include <string>
 #include <fstream>
 #include <iostream>
+#include <cstring>
 
 using Atlas::Message::Element;
 using Atlas::Message::MapType;
