diff -urpN entityforge-0.2.old/configure entityforge-0.2/configure
--- entityforge-0.2.old/configure	2004-10-09 14:03:59.514660984 -0700
+++ entityforge-0.2/configure	2004-10-09 14:05:50.011862856 -0700
@@ -10850,23 +10850,23 @@ fi
   else
      PKG_CONFIG_MIN_VERSION=0.9.0
      if $PKG_CONFIG --atleast-pkgconfig-version $PKG_CONFIG_MIN_VERSION; then
-        echo "$as_me:$LINENO: checking for gtkmm-2.0 >= 2.2.2" >&5
-echo $ECHO_N "checking for gtkmm-2.0 >= 2.2.2... $ECHO_C" >&6
+        echo "$as_me:$LINENO: checking for gtkmm-2.4 >= 2.4.0" >&5
+echo $ECHO_N "checking for gtkmm-2.4 >= 2.4.0... $ECHO_C" >&6
 
-        if $PKG_CONFIG --exists "gtkmm-2.0 >= 2.2.2" ; then
+        if $PKG_CONFIG --exists "gtkmm-2.4 >= 2.4.0" ; then
             echo "$as_me:$LINENO: result: yes" >&5
 echo "${ECHO_T}yes" >&6
             succeeded=yes
 
             echo "$as_me:$LINENO: checking GTKMM_CFLAGS" >&5
 echo $ECHO_N "checking GTKMM_CFLAGS... $ECHO_C" >&6
-            GTKMM_CFLAGS=`$PKG_CONFIG --cflags "gtkmm-2.0 >= 2.2.2"`
+            GTKMM_CFLAGS=`$PKG_CONFIG --cflags "gtkmm-2.4 >= 2.4.0"`
             echo "$as_me:$LINENO: result: $GTKMM_CFLAGS" >&5
 echo "${ECHO_T}$GTKMM_CFLAGS" >&6
 
             echo "$as_me:$LINENO: checking GTKMM_LIBS" >&5
 echo $ECHO_N "checking GTKMM_LIBS... $ECHO_C" >&6
-            GTKMM_LIBS=`$PKG_CONFIG --libs "gtkmm-2.0 >= 2.2.2"`
+            GTKMM_LIBS=`$PKG_CONFIG --libs "gtkmm-2.4 >= 2.4.0"`
             echo "$as_me:$LINENO: result: $GTKMM_LIBS" >&5
 echo "${ECHO_T}$GTKMM_LIBS" >&6
         else
@@ -10874,7 +10874,7 @@ echo "${ECHO_T}$GTKMM_LIBS" >&6
             GTKMM_LIBS=""
             ## If we have a custom action on failure, don't print errors, but
             ## do set a variable so people can do so.
-            GTKMM_PKG_ERRORS=`$PKG_CONFIG --errors-to-stdout --print-errors "gtkmm-2.0 >= 2.2.2"`
+            GTKMM_PKG_ERRORS=`$PKG_CONFIG --errors-to-stdout --print-errors "gtkmm-2.4 >= 2.4.0"`
             echo $GTKMM_PKG_ERRORS
         fi
 
@@ -10889,8 +10889,8 @@ echo "${ECHO_T}$GTKMM_LIBS" >&6
   if test $succeeded = yes; then
      :
   else
-     { { echo "$as_me:$LINENO: error: Library requirements (gtkmm-2.0 >= 2.2.2) not met; consider adjusting the PKG_CONFIG_PATH environment variable if your libraries are in a nonstandard prefix so pkg-config can find them." >&5
-echo "$as_me: error: Library requirements (gtkmm-2.0 >= 2.2.2) not met; consider adjusting the PKG_CONFIG_PATH environment variable if your libraries are in a nonstandard prefix so pkg-config can find them." >&2;}
+     { { echo "$as_me:$LINENO: error: Library requirements (gtkmm-2.4 >= 2.4.0) not met; consider adjusting the PKG_CONFIG_PATH environment variable if your libraries are in a nonstandard prefix so pkg-config can find them." >&5
+echo "$as_me: error: Library requirements (gtkmm-2.4 >= 2.4.0) not met; consider adjusting the PKG_CONFIG_PATH environment variable if your libraries are in a nonstandard prefix so pkg-config can find them." >&2;}
    { (exit 1); exit 1; }; }
   fi
 
diff -urpN entityforge-0.2.old/configure.ac entityforge-0.2/configure.ac
--- entityforge-0.2.old/configure.ac	2004-10-09 14:03:59.518660376 -0700
+++ entityforge-0.2/configure.ac	2004-10-09 14:05:56.275910576 -0700
@@ -13,7 +13,7 @@ AM_PROG_LIBTOOL
 
 AC_LANG_CPLUSPLUS
 
-PKG_CHECK_MODULES(GTKMM,[gtkmm-2.0 >= 2.2.2])
+PKG_CHECK_MODULES(GTKMM,[gtkmm-2.4 >= 2.4.0])
 AC_SUBST(GTKMM_CFLAGS)
 AC_SUBST(GTKMM_LIBS)
 
