diff -Naur sauerbraten.orig/src/shared/tools.cpp sauerbraten/src/shared/tools.cpp
--- sauerbraten.orig/src/shared/tools.cpp	2005-10-03 20:21:00.000000000 -0700
+++ sauerbraten/src/shared/tools.cpp	2005-10-03 20:23:36.000000000 -0700
@@ -5,8 +5,24 @@
 
 ///////////////////////// misc tools ///////////////////////
 
+// Modified from http://bugs.gentoo.org/show_bug.cgi?id=104292
+char *addfullpath ( char *s )
+{
+  static char ret[256];      // choose a reasonable max buffer size
+  if (s[0] == '/')
+  {
+    return s;
+  }
+  int slen = strlen(s);
+  memset(ret, 0x00, 256);
+  memcpy(ret, GAMES_DATADIR, GAMES_DATADIR_LEN);
+  memcpy(ret+GAMES_DATADIR_LEN, s, slen);
+  return ret;
+};
+
 char *path(char *s)
 {
+    s = addfullpath ( s );
     for(char *t = s; t = strpbrk(t, "/\\"); *t++ = PATHDIV);
     return s;
 };
@@ -91,3 +107,4 @@
     y ^= (y << 15) & 0xEFC60000U;
     return(y ^ (y >> 18));
 };
+
