diff -Naur sauerbraten.orig/src/engine/rendergl.cpp sauerbraten/src/engine/rendergl.cpp
--- sauerbraten.orig/src/engine/rendergl.cpp	2005-10-03 21:51:10.000000000 -0700
+++ sauerbraten/src/engine/rendergl.cpp	2005-10-03 21:56:48.000000000 -0700
@@ -116,6 +116,8 @@
 
 Texture *textureload(char *name, int rot, bool clamp, bool mipit, bool msg)
 {
+    name = addfullpath ( name );
+
     string rname, tname;
     strcpy_s(tname, name);
     strcpy_s(rname, path(tname));
