diff -Naur sauerbraten.orig/src/shared/tools.h sauerbraten/src/shared/tools.h
--- sauerbraten.orig/src/shared/tools.h	2005-10-03 21:42:47.000000000 -0700
+++ sauerbraten/src/shared/tools.h	2005-10-03 21:46:41.000000000 -0700
@@ -115,6 +115,7 @@
 #endif
 
 
+extern char *addfullpath ( char* s );
 extern char *path(char *s);
 extern char *loadfile(char *fn, int *size);
 extern void endianswap(void *, int, int);
