diff -urpN kaf.orig/configure kaf/configure
--- kaf.orig/configure	2004-10-18 16:59:55.103350960 +0000
+++ kaf/configure	2004-10-18 17:00:51.968706112 +0000
@@ -52,9 +52,9 @@ else
 fi
 
 echo -n "Looking for QT include files... "
-if ! test -e "$QTDIR/include/qapplication.h"; then
+if ! test -e "$QTDIR/include/qt/qapplication.h"; then
   echo "not found."
-  echo "Error: Cannot find '$QTDIR/include/qapplication.h'."
+  echo "Error: Cannot find '$QTDIR/include/qt/qapplication.h'."
   exit 1
 else
   echo "found."
