--- mkres/convert.h~	2007-04-26 16:19:34.000000000 +0000
+++ mkres/convert.h	2007-04-26 16:19:42.000000000 +0000
@@ -5,6 +5,7 @@
 #include <iostream>
 #include <sstream>
 #include <string>
+#include <typeinfo>

 #include "exceptions.h"
 #include "unicode.h"
--- convert.h~	2007-04-26 16:20:45.000000000 +0000
+++ convert.h	2007-04-26 16:20:56.000000000 +0000
@@ -5,6 +5,7 @@
 #include <iostream>
 #include <sstream>
 #include <string>
+#include <typeinfo>

 #include "exceptions.h"
 #include "unicode.h"
--- einstein-2.0.dfsg.2/unicode.cpp	2005-08-14 04:40:58.000000000 +0200
+++ mkres/unicode.cpp	2008-01-26 11:49:10.000000000 +0100
@@ -1,5 +1,6 @@
 #include <wchar.h>
 #include <stdlib.h>
+#include <cstring>
 #ifdef WIN32
 #include <windows.h>
 #endif
--- einstein-2.0.dfsg.2/unicode.cpp	2005-08-14 04:40:58.000000000 +0200
+++ unicode.cpp	2008-01-26 11:49:10.000000000 +0100
@@ -1,5 +1,6 @@
 #include <wchar.h>
 #include <stdlib.h>
+#include <cstring>
 #ifdef WIN32
 #include <windows.h>
 #endif
--- compressor.cpp.orig     2008-06-13 16:19:23.000000000 +0200
+++ mkres/compressor.cpp  2008-06-13 16:19:33.000000000 +0200
@@ -2,7 +2,7 @@
 #include <zlib.h>
 #include "convert.h"
 #include "exceptions.h"
-
+#include <cstring>

 ResourceCompressor::ResourceCompressor()
 {
--- main.cpp.o    2008-06-13 16:28:50.000000000 +0200
+++ mkres/main.cpp        2008-06-13 16:29:09.000000000 +0200
@@ -1,3 +1,4 @@
+#include <cstring>
 #include "compressor.h"
 #include "exceptions.h"
 #include "unicode.h"

