--- src/Makefile.in.orig	2008-02-03 11:49:47.000000000 +0500
+++ src/Makefile.in	2008-02-03 11:51:05.000000000 +0500
@@ -2468,21 +2468,7 @@
 @WIN32_TRUE@.ico.o:
 @WIN32_TRUE@	echo "1 ICON \"""$<""\"" > tmp.rc && i586-mingw32msvc-windres tmp.rc "$@"
 
-# Force svn_version.o to always be remade if necessary
-all: svnCheck
-	@echo svnCheck
 
-svnCheck:
-	if test -d $(srcdir)/.svn; then \
-	  if ! test -f svnVersion -a "`cat svnVersion`" = "`svnversion -n $(srcdir)`"; then \
-	    svnversion -n $(srcdir) > svnVersion; \
-	  fi; \
-	fi
-
-svnVersion:
-	 svnversion -n $(srcdir) > svnVersion
-
-svn_version.$(OBJEXT): svnVersion
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
 .NOEXPORT:
