diff -Nur mahjongg3d.release/Makefile mahjongg3d.release.fix/Makefile
--- mahjongg3d.release/Makefile	2004-05-13 14:17:23.000000000 +0200
+++ mahjongg3d.release.fix/Makefile	2004-05-15 11:36:00.000000000 +0200
@@ -25,7 +25,7 @@
 sub-src: src/$(MAKEFILE) FORCE
 	cd src && $(MAKE) -f $(MAKEFILE)
 
-Makefile: mahjongg3d.pro  $(QTDIR)/mkspecs/default/qmake.conf 
+Makefile: mahjongg3d.pro
 	$(QMAKE) -o Makefile mahjongg3d.pro
 qmake: qmake_all
 	@$(QMAKE) -o Makefile mahjongg3d.pro
