diff -Nur mahjongg3d.release/Makefile.doit mahjongg3d.release.fix/Makefile.doit
--- mahjongg3d.release/Makefile.doit	2004-05-28 17:32:41.000000000 +0200
+++ mahjongg3d.release.fix/Makefile.doit	2004-06-02 17:22:32.826061836 +0200
@@ -13,7 +13,7 @@
 MKDIR    = mkdir -p
 SUBTARGETS =	 \
 		sub-src
-PREFIX=~
+PREFIX=/usr/share
 GAMEDATA_PREFIX=$(PREFIX)
 
 first: all
@@ -32,7 +32,7 @@
 	echo \" >> gamedata_path.h && \
 	$(MAKE) -f $(MAKEFILE) 
 
-Makefile: mahjongg3d.pro  $(QTDIR)/mkspecs/default/qmake.conf 
+Makefile: mahjongg3d.pro
 	$(QMAKE) -o Makefile mahjongg3d.pro
 qmake: qmake_all
 	@$(QMAKE) -o Makefile mahjongg3d.pro
