--- engine/stratagus/stratagus.cpp	2008-03-01 21:15:36.000000000 +0100
+++ engine/stratagus/stratagus2.cpp	2008-09-27 23:11:53.000000000 +0200
@@ -160,6 +160,9 @@
 #include <time.h>
 #include <ctype.h>
 #include <sstream>
+#include <algorithm>
+#include <streambuf>
+#include <vector>
 
 #ifndef _MSC_VER
 #include <unistd.h>
--- engine/pathfinder/astar.cpp	2008-03-01 21:15:35.000000000 +0100
+++ engine/pathfinder/astar2.cpp 2008-09-27 22:15:30.000000000 +0200
@@ -32,7 +32,7 @@
 /*----------------------------------------------------------------------------
 --  Includes
 ----------------------------------------------------------------------------*/
-
+#include "stdlib.h"
 #include "stratagus.h"
 #include "pathfinder.h"
 
