diff -ruN Chromium-0.9.old/config.mak Chromium-0.9/config.mak
--- Chromium-0.9.old/config.mak	1970-01-01 10:00:00.000000000 +1000
+++ Chromium-0.9/config.mak	2003-06-14 13:23:18.000000000 +1000
@@ -0,0 +1,14 @@
+OPENAL_CONFIG_OPTS = ./configure --enable-sdl --enable-smpeg --enable-vorbis
+
+SDL_CONFIG   = /usr/bin/sdl-config
+SMPEG_CONFIG = /usr/bin/smpeg-config
+
+SDL_CFLAGS   = `$(SDL_CONFIG) --cflags` -DUSE_SDL
+SMPEG_CFLAGS = `$(SMPEG_CONFIG) --cflags`
+PKG_CFLAGS   = 
+
+GL_LIBS      = -lglpng -lGL -lGLU -lm
+AL_LIBS      = -L../support/openal/lib -lopenal -ldl
+SDL_LIBS     = `$(SDL_CONFIG) --libs`
+SMPEG_LIBS   = `$(SMPEG_CONFIG) --libs`
+VORBIS_LIBS  = -L/usr/lib -lvorbis -lvorbisfile
diff -ruN Chromium-0.9.old/Makefile Chromium-0.9/Makefile
--- Chromium-0.9.old/Makefile	1970-01-01 10:00:00.000000000 +1000
+++ Chromium-0.9/Makefile	2003-06-14 13:22:57.000000000 +1000
@@ -0,0 +1,48 @@
+SUPPORT_DIRS	=	 \
+			./support/openal/ \
+			./support/glpng/src
+
+CHROMIUM_DIRS	=	\
+			./src
+
+			
+all: $(SUPPORT_DIRS) chromium
+
+chromium: $(CHROMIUM_DIRS) finish
+
+support: $(SUPPORT_DIRS)
+
+$(CHROMIUM_DIRS): FORCE
+	cd $@; $(MAKE)
+
+$(SUPPORT_DIRS): FORCE
+	cd $@; $(MAKE)
+
+finish: 
+	@echo ""
+	@mv -vf ./src/chromium ./bin
+	@echo ""
+	-ls --color ./bin/*
+	@echo ""
+	@echo "Now cd to ./bin and run chromium!"
+	@echo ""
+
+clean:
+	for i in $(SUPPORT_DIRS); do ( cd $$i ; $(MAKE) clean ) ; done
+	for i in $(CHROMIUM_DIRS); do ( cd $$i ; $(MAKE) clean ) ; done
+	-rm -f ./bin/*
+	-rm -f *~ core
+	-rm -f *.bck
+
+distclean: clean
+	@echo ""
+	-rm -f scripts/*.bck
+	@echo -e "\n ** freshen up the doc files **\n--------------------------------"
+	-cp -f /home/mallan/HTML/mba/software/chromium/faq.htm ./data/doc
+	-cp -f /home/mallan/HTML/mba/software/chromium/info.htm ./data/doc
+	@echo ""
+	-rm -f ./config.mak
+	-rm -f ./Makefile
+	@echo ""
+
+FORCE:
diff -ruN Chromium-0.9.old/src/Makefile Chromium-0.9/src/Makefile
--- Chromium-0.9.old/src/Makefile	2001-05-22 15:25:50.000000000 +1000
+++ Chromium-0.9/src/Makefile	2003-06-14 14:30:46.000000000 +1000
@@ -10,8 +10,8 @@
 
 CC	=	gcc
 CXX	=	g++
-CFLAGS	=	-pipe $(PKG_CFLAGS) $(AL_CFLAGS) $(SDL_CFLAGS) $(SMPEG_CFLAGS) -O2 -DOLD_OPENAL -DAUDIO_OPENAL -D_REENTRANT
-CXXFLAGS=	-pipe $(PKG_CFLAGS) $(AL_CFLAGS) $(SDL_CFLAGS) $(SMPEG_CFLAGS) -O2 -DOLD_OPENAL -DAUDIO_OPENAL -D_REENTRANT
+CFLAGS	=	-pipe $(PKG_CFLAGS) $(AL_CFLAGS) $(SDL_CFLAGS) $(SMPEG_CFLAGS) -O2 -DAUDIO_OPENAL -D_REENTRANT $(OLDCFLAGS)
+CXXFLAGS=	-pipe $(PKG_CFLAGS) $(AL_CFLAGS) $(SDL_CFLAGS) $(SMPEG_CFLAGS) -O2 -DAUDIO_OPENAL -D_REENTRANT $(OLDCXXFLAGS)
 INCPATH	=	-I../support/include -I../support/glpng/include -I/usr/X11R6/include
 LINK	=	g++
 LFLAGS	=	
diff -ruN Chromium-0.9.old/src/NCString.h Chromium-0.9/src/NCString.h
--- Chromium-0.9.old/src/NCString.h	2001-05-21 03:05:23.000000000 +1000
+++ Chromium-0.9/src/NCString.h	2003-06-14 13:23:46.000000000 +1000
@@ -2,6 +2,7 @@
 #define NCString_h
 
 #include <stream.h>
+#include <tcl.h>
 
 typedef unsigned int uint;
 typedef unsigned short ushort;
diff -ruN Chromium-0.9.old/support/openal/Makefile Chromium-0.9/support/openal/Makefile
--- Chromium-0.9.old/support/openal/Makefile	2001-02-08 16:09:44.000000000 +1100
+++ Chromium-0.9/support/openal/Makefile	2003-06-14 13:24:02.000000000 +1000
@@ -4,7 +4,7 @@
 
 linux: FORCE
 	mkdir -p ./lib
-	cd linux; sh autogen.sh; $(OPENAL_CONFIG_OPTS); $(MAKE)
+	cd linux; $(OPENAL_CONFIG_OPTS); $(MAKE)
 	cp ./linux/src/libopenal.a ./lib
 
 clean:
