--- lotrmap/configfile.cpp	2004-07-14 20:54:39.439573768 -0700
+++ lotrmap/configfile.cpp	2004-07-14 20:56:08.812986944 -0700
@@ -200,7 +200,7 @@
 	return newSection;
 }
 
-bool ConfigFile::findSection( const QString &section, Section **thesection, bool create=true )
+bool ConfigFile::findSection( const QString &section, Section **thesection, bool create )
 {
 	Section * s;
 	for( s = sectionList.first(); s != 0; s = sectionList.next() ) {
