diff -ru dosbox-0.74/src/hardware/pic.cpp dosbox-0.74-epic/src/hardware/pic.cpp
--- dosbox-0.74/src/hardware/pic.cpp	2010-05-10 19:43:54.000000000 +0200
+++ dosbox-0.74-epic/src/hardware/pic.cpp	2012-05-19 15:28:56.000000000 +0200
@@ -541,12 +541,12 @@
 }
 
 
-class PIC:public Module_base{
+class EPIC:public Module_base{
 private:
 	IO_ReadHandleObject ReadHandler[4];
 	IO_WriteHandleObject WriteHandler[4];
 public:
-	PIC(Section* configuration):Module_base(configuration){
+	EPIC(Section* configuration):Module_base(configuration){
 		/* Setup pic0 and pic1 with initial values like DOS has normally */
 		PIC_IRQCheck=0;
 		PIC_IRQActive=PIC_NOIRQ;
@@ -596,17 +596,17 @@
 		pic_queue.free_entry=&pic_queue.entries[0];
 		pic_queue.next_entry=0;
 	}
-	~PIC(){
+	~EPIC(){
 	}
 };
 
-static PIC* test;
+static EPIC* test;
 
 void PIC_Destroy(Section* sec){
 	delete test;
 }
 
 void PIC_Init(Section* sec) {
-	test = new PIC(sec);
+	test = new EPIC(sec);
 	sec->AddDestroyFunction(&PIC_Destroy);
 }
