--- scorched/src/common/SoundStore.h~	2003-07-10 17:28:58.000000000 +0200
+++ scorched/src/common/SoundStore.h	2003-07-30 18:57:12.000000000 +0200
@@ -29,7 +29,7 @@
 #include <string>
 
 #define CACHE_SOUND(var, filename) 										\
-		static SoundBuffer* ##var = SoundStore::instance()->fetchOrCreateBuffer(##filename);
+		static SoundBuffer* var = SoundStore::instance()->fetchOrCreateBuffer(filename);
 
 class SoundStore
 {
--- scorched/src/engine/ActionMeta.h~	2003-07-10 17:29:10.000000000 +0200
+++ scorched/src/engine/ActionMeta.h	2003-07-30 18:49:55.000000000 +0200
@@ -29,7 +29,7 @@
 
 #define REGISTER_ACTION_HEADER(x) \
 	virtual const char *getActionName() { return #x ; } \
-	##x *getActionCopy() { return new x ; } \
+	x *getActionCopy() { return new x ; } \
 	virtual bool getReferenced() { return true; }
 #define REGISTER_ACTION_SOURCE(x) \
 	struct META_##x { META_##x() { ActionMetaRegistration::addMap(#x , new x ); } }; \
